src/codegeneration/ParenTrait.js

```diff
@@ -129,5 +130,16 @@ export function ParenTrait(ParseTreeTransformerClass) {
       }
       return new ArrayLiteralExpression(tree.location, elements);
     }
+
+    transformPropertyNameAssignment(tree) {
+      let name = this.transformAny(tree.name);
+      let value = this.transformAny(tree.value);
+      if (value.type === COMMA_EXPRESSION) {
```

Comment:
No. COMMA_EXPRESSION is NOT an AssignmentExpression.

We could maybe change our code to something like `tree.isAssignmentExpression()`